lib/inspec/resources/shadow.rb

```diff
@@ -52,12 +52,6 @@ def initialize(path = "/etc/shadow", opts = {})
       .register_column(:inactive_days, field: "inactive_days")
       .register_column(:expiry_dates, field: "expiry_date")
       .register_column(:reserved, field: "reserved")
-    # These are deprecated, but we need to "alias" them
-    filtertable
-      .register_custom_property(:user) { |table, value| table.resource.user(value) }
-      .register_custom_property(:password) { |table, value| table.resource.password(value) }
-      .register_custom_property(:last_change) { |table, value| table.resource.last_change(value) }
-      .register_custom_property(:expiry_date) { |table, value| table.resource.expiry_date(value) }
```

Comment:
@james-stocks can we get confirmation it is safe to remove these at this point?

Reply:
These are what let you directly call like `shadow.user`?

All our supported profiles use `shadow.where` and we don't need anything other than `where`.

Reply:
There were no aliases. There were duplicates. I removed the former, not the latter. All the method names are still there. If I removed names, we'd have test failures (I hope).